incrementals-publisher/index.js

```diff
@@ -52,7 +52,7 @@ module.exports = async (context, data) => {
    * The first step is to take the buildUrl and fetch some metadata about this
    * specific Pipeline Run
    */
-  let metdata_url = pipeline.getApiUrl(buildUrl);
+  let metdataUrl = pipeline.getApiUrl(buildUrl);
```

Comment:
Still a typo.

Gee this dynamic language thing is great! What was I so worried about?